src/main/app/forms/validation/validators/atLeastOnePopulatedRow.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface
+} from 'class-validator'
+
+import { MultiRowForm } from 'forms/models/multiRowForm'
+import { MultiRowFormItem } from 'forms/models/multiRowFormItem'
+
+@ValidatorConstraint()
+export class AtLeastOnePopulatedRowConstraint implements ValidatorConstraintInterface {
+
+  validate (value: any | MultiRowForm<MultiRowFormItem>, args?: ValidationArguments): boolean {
+    if (value == null) {
```

Comment:
null?

Reply:
we have line like this in all validators...

Reply:
Untrue, I checked most of them and approximately half (including the first 3 I checked from the top) all use undefined.
New code should follow the newer convention, and if you would like to update all of them afterwards great too